WooCommerce/Classes/POS/Presentation/PaymentButtons.swift

```diff
@@ -65,7 +65,9 @@ private extension PaymentsActionButtons {
     func handleSendReceiptAction() async {
         let isEligible = await checkReceiptEligibility()
         if isEligible {
-            isShowingSendReceiptView = true
+            withAnimation {
```

Comment:
I got different animation behaviour using `withAnimation` vs `.animation(value:)`. I'm not entirely sure why, but `withAnimation` seems to work better despite I like less how we have to call it. For context I've tested this with iOS 17

Reply:
I suppose it can depend on where we put `.animation(value:)` in a view hierarchy but I haven't developed a good intuition yet on a precise way of working.

Reply:
I'm not sure how to explain it, is the same animation but looks more clunky 😅 ,definitely tricky to choose where to put it. I've removed this explicit call here: 4a392e33ea5b8d94301b0f46ca1ce511a77da5ce